sympy/integrals/transforms.py

```diff
@@ -1150,7 +1155,37 @@ class LaplaceTransform(IntegralTransform):
 
     _name = 'Laplace'
 
+    def _cr(self, f, a, c, **hints):
+        conds = not hints.get('noconds', False)
+        if conds:
+            return f, a, c
+        else:
+            return f
+
     def _compute_transform(self, f, t, s, **hints):
+        a = Wild('a', exclude=[t])
+        b = Wild('b', exclude=[t])
+        c = Wild('c', exclude=[t])
+        tau = Wild('tau', exclude=[t])
+        omega = Wild('omega', exclude=[t])
+        simple_rules = [
+            # ( time domain, laplace domain, condition, convergence plane )
+            (b*exp(a*t-tau), b*exp(-tau)/(s-a), S.true, a), # 3.1
+            (b*t*exp(a*t-tau), b*exp(-tau)/(s-a)**2, S.true, a), # 3.2
+            (b*t**c*exp(a*t), b*gamma(c+1)/(s-a)**(c+1), c>-1, a), # 3.3
+            (a*sin(omega*t), a*omega/(s**2+omega**2), S.true, S.Zero), #6.1
+            (a*sin(omega*t)/t, a*atan(omega/s), S.true, S.Zero), # 6.6
+            (a*cos(omega*t), a*s/(s**2+omega**2), S.true, S.Zero), # 6.11
+            (c*exp(b*t)*sin(a*t), c*a/((s-b)**2+a**2), S.true, b), # 6.21
+            (c*exp(b*t)*cos(a*t), c*(s-b)/((s-b)**2+a**2), S.true, b), # 6.22
+            ]
+        for rule in simple_rules:
```

Comment:
Maybe one would like to unpack the rule here, like:
```
for time_domain, laplace_domain, condition, convergence_plane in simple_rules:
```
(or something more compact).

Will maybe make the code a bit easier to read.